src/main/java/ladder/model/Ladder.java

```diff
@@ -0,0 +1,59 @@
+package ladder.model;
+
+import ladder.constant.LadderPath;
+import ladder.dto.LineDto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Ladder {
+    private static final Random random = new Random();
+    private final List<Line> ladder;
+
+    private Ladder(List<Line> ladder) {
+        this.ladder = ladder;
+    }
+
+    public static Ladder of(LadderSize ladderSize) {
+        List<Line> ladder = new ArrayList<>();
+        int height = ladderSize.height();
+        int width = ladderSize.width();
+
+        for (int i = 0; i < height; i++) {
+            ladder.add(new Line(makeRandomRow(width)));
+        }
+
+        return new Ladder(ladder);
```

Comment:
```suggestion
        int height = ladderSize.height();
        int width = ladderSize.width();

        List<Line> ladder = new ArrayList<>();
        for (int i = 0; i < height; i++) {
            ladder.add(new Line(makeRandomRow(width)));
        }

        return new Ladder(ladder);
```
지역변수는 웬만하면 사용하는 곳에서 최대한 가까운 위치에 초기화 하는 것을 추천해요.
훨씬 로직을 읽기도 편해지고 관리하기도 좋거든요!

Reply:
C 스타일이 익숙해서 그런지 별 생각 없이 항상 메서드 최상단에서 모든 지역변수를 초기화 하고 로직을 작성했었네요ㅋㅋㅋ 조언 감사합니다!

Reply:
이 코멘트를 좀 보충하자면 height, width 도 지역변수이긴 한데, 변경되는 요소를 변경점과 가깝게 두자는 말이었어요 ㅋㅋㅋ
페드로가 잘 이해해주셨군요! 👍